content/ko/docs/reference/glossary/annotation.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 어노테이션(Annotation)
+title: 어노테이션
```

Comment:
```suggestion
title: 어노테이션(Annotation)
```
We intentionally provide the original text in the title of each file in `/ko/docs/reference/glossary/*`
Because some readers want to know the original text for terms in K8s glossary.

So, please restore them :)

Reply:
please restore all changes in `/ko/docs/reference/glossary/*`